packages/angular-teleport/src/lib/teleport.service.ts

```diff
@@ -0,0 +1,35 @@
+import {
+  ApplicationRef,
+  ComponentFactoryResolver,
+  ComponentRef,
+  Injectable,
+  Injector,
+  Type,
+} from '@angular/core';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class TeleportService {
+  public constructor(
+    private readonly appRef: ApplicationRef,
+    private readonly componentFactoryResolver: ComponentFactoryResolver,
+    private readonly injector: Injector
+  ) {}
+
+  public create<T>(
+    target: HTMLElement,
```

Comment:
maybe make this optional as well and fall back to `document.body` like in the directive.

Reply:
Fixed.